Source/Diagnostics/WarpXOpenPMD.H

```diff
@@ -152,6 +152,13 @@ public:
   /** Return OpenPMD File type ("bp5", "bp4", "h5" or "json")*/
   std::string OpenPMDFileType () { return m_OpenPMDFileType; }
 
+  /** Flush a few BTD buffers in a snapshot
+   * @param[in] isBTD if the current diagnostic is BTD
+   * This function is controlled by the paramter
+   * FlushFormatOpenPMD::m_NumAggBTDBufferToFlush (default to 5),
+   * it can be adjusted in the input file: diag_name.buffer_flush_limit_btd
```

Comment:
Making this valid Doxygen:
```suggestion
   *
   * This function is controlled by the parameter
   * FlushFormatOpenPMD::m_NumAggBTDBufferToFlush (default to 5),
   * it can be adjusted in the input file: diag_name.buffer_flush_limit_btd
   *
   * @param[in] isBTD if the current diagnostic is BTD
```